tsl/src/compression/compress_utils.c

```diff
@@ -552,20 +552,22 @@ tsl_recompress_remote_chunk(Chunk *uncompressed_chunk, FunctionCallInfo fcinfo)
 	FunctionCallInvoke(decompfn_fcinfo);
 	if (decompfn_fcinfo->isnull)
 	{
-		ereport(WARNING,
+		ereport((if_compressed ? NOTICE : ERROR),
 				(errcode(ERRCODE_INTERNAL_ERROR),
-				 errmsg("decompression failed for recompress_chunk for %u",
-						uncompressed_chunk->fd.id)));
+				 errmsg("decompression failed for recompress_chunk for %u status=%d",
+						uncompressed_chunk->fd.id,
+						uncompressed_chunk->fd.status)));
 		PG_RETURN_NULL();
 	}
 	get_compression_fcinfo(COMPRESS_CHUNK_FUNCNAME, &compfn, &compfn_fcinfo, fcinfo);
 	Datum compoid = FunctionCallInvoke(compfn_fcinfo);
 	if (compfn_fcinfo->isnull)
 	{
-		ereport(WARNING,
+		ereport((if_compressed ? NOTICE : ERROR),
 				(errcode(ERRCODE_INTERNAL_ERROR),
-				 errmsg("compression failed for recompress_chunk for %u",
-						uncompressed_chunk->fd.id)));
+				 errmsg("compression failed for recompress_chunk for %u with status=%d",
```

Comment:
Same here as above (for decompress case)

Reply:
fixed.